src/backend/aspen/workflows/nextstrain_run/run_nextstrain_scheduled.sh

```diff
@@ -59,6 +61,9 @@ aligned_gisaid_location=$(
            --builds-file /ncov/my_profiles/aspen/builds.yaml       \
 )
 
+# temporary fix for recency focused automatic build
+# even if the group has no samples in the past 3 months, the run will not fail
+sed -i '/include\.txt/d' /ncov/my_profiles/aspen/builds.yaml
```

Comment:
this will make the run use default `include.txt` from `ncov`. if the county has no samples from past 3 months, the tree will be built around sequences in the default `include.txt` (and will not be very useful XD). I put it here instead of yaml builder since it's only applicable to scheduled runs.

Reply:
@jgadling this should make all automatic overview tree work.